src/panels/iframe/ha-panel-iframe.ts

```diff
@@ -41,7 +41,9 @@ class HaPanelIframe extends LitElement {
           )}
           src=${this.panel.config.url}
           sandbox="allow-forms allow-popups allow-pointer-lock allow-same-origin allow-scripts allow-modals allow-downloads"
-          allow="fullscreen"
+          allow=${ifDefined(
+            this.panel.config.allow ?? 'fullscreen'
+          )}
```

Comment:
I don't think `ifDefined` is needed as the value of the attr will always be defined, either by `config.allow` or by the default value "fullscreen"

```suggestion
          allow=${this.panel.config.allow ?? 'fullscreen'}
```